servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultRequestTracker.java

```diff
@@ -155,7 +158,7 @@ public final int score() {
         // Add penalty for pending requests to account for "unaccounted" load.
         // Penalty is the observed latency if known, else an arbitrarily high value which makes entities for which
         // no latency data has yet been received (eg: request sent but not received), un-selectable.
-        final int pendingPenalty = (int) min(MAX_VALUE, (long) cPending * currentEWMA);
+        final int pendingPenalty = (int) min(MAX_VALUE, (long) cPending * pendingRequestPenalty * currentEWMA);
```

Comment:
This would arguably be better as a double since most useful values of this will center around the low single digits, most likely 0 and 1 being the most common. Otoh, maybe it's not worth letting people try to overthink it to find if 0.5 is better than 1. Feedback welcome.

Reply:
1. What range is currently expected for resulting `pendingPenalty`?
2. Why do we use `long` for config parameters but cast it to `int` internally?

Reply:
1. realistic values range between 0 and probably 100 where 100 is effectively infinite.
2. I used a long for consistency with other parameters such as `errorPenalty` and `cancelPenalty`, which could arguably be turned to ints themselves.

Reply:
I'm happy to switch from long to int but would prefer to do the same with the cancellation and error penalties if we do. That should make Integer.MAX_VALUE safe to use as without worrying about overflows. However, I suspect we'd want to go through a deprecation cycle for the `long` method.

Reply:
int is easier to operate with. If there is no need to have values larger than `Integer.MAX_VALUE`, +1 for changing to longs. if you are sure nobody used `long` methods outside our providers, simply change the type. if there is a risk, we need to go through deprecation